src/parser.cpp

```diff
@@ -36,4 +37,118 @@ types::VPDMapVariant Parser::parse()
     return l_parser->parse();
 }
 
+int Parser::updateVpdKeyword(const types::WriteVpdParams& i_paramsToWriteData)
+{
+    // Update keyword's value on hardware
+    int l_bytesUpdatedOnHardware;
+    try
+    {
+        std::shared_ptr<ParserInterface> l_vpdParserInstance =
+            getVpdParserInstance();
+        l_bytesUpdatedOnHardware =
+            l_vpdParserInstance->writeKeywordOnHardware(i_paramsToWriteData);
+    }
+    catch (const std::exception& l_exception)
+    {
+        // TODO : Log PEL
```

Comment:
Please log to console as well in case of error scenarios.
It will be easy to debug, else during test phase it will be very tricy to go to PEL always and match with time stamp.

Reply:
done